docs/sql-programming-guide.md

```diff
@@ -1258,7 +1258,12 @@ This command builds a new assembly jar that includes Hive. Note that this Hive a
 on all of the worker nodes, as they will need access to the Hive serialization and deserialization libraries
 (SerDes) in order to access data stored in Hive.
 
-Configuration of Hive is done by placing your `hive-site.xml` file in `conf/`.
+Configuration of Hive is done by placing your `hive-site.xml` file in `conf/`. Please note when running 
+the query on YARN cluster (`yarn-cluster` mode),the `datanucleus` jars under `lib_managed/jars` directory 
```

Comment:
comma needs a space after, add "the" before spark-submit and put it in `code font`.